05week/spaceTravelToMars.js

```diff
@@ -10,6 +10,33 @@ let jobTypes = {
 };
 
 // Your code here
+class crewMember {
+  constructor(name, job, specialSkill, ship) {
+    this.name = name;
+    this.job = job;
+    this.specialSkill = specialSkill;
+    this.ship = ship;
```

Comment:
when we construct the crew member we should not pass the ship. The ship gets set by the "enterShip" method.